include/internal/catch_common.hpp

```diff
@@ -22,7 +24,7 @@ namespace Catch {
         return s.find( infix ) != std::string::npos;
     }
     void toLowerInPlace( std::string& s ) {
-        std::transform( s.begin(), s.end(), s.begin(), ::tolower );
+        std::transform( s.begin(), s.end(), s.begin(), static_cast<int(*)(int)>(std::tolower) );
```

Comment:
There's also `charT tolower( charT ch, const locale& loc );` in `<locale>`, so we're better off using a lambda anyway